projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -590,6 +609,13 @@ public static function validate_task( $task ) {
 			return false;
 		}
 
+		$missing_repetition_property = ( isset( $task['target_repetitions'] ) || isset( $task['repetition_count_callback'] ) ) && ! ( isset( $task['target_repetitions'] ) && isset( $task['repetition_count_callback'] ) );
+
+		if ( $missing_repetition_property ) {
```

Comment:
This logic is pretty complicated. Might it be clearer if we split the logic up into two distinct checks?
```suggestion
		$has_any_repetition_properties  = isset( $task['target_repetitions'] ) || isset( $task['repetition_count_callback'] );
		$has_both_repetition_properties = isset( $task['target_repetitions'] ) && isset( $task['repetition_count_callback'] );

		if ( $has_any_repetition_properties && ! $has_both_repetition_properties ) {
```

Reply:
Another more concise way of putting it might be:
```
$missing_repetition_property = isset( $task['target_repetitions'] ) xor isset( $task['repetition_count_callback'] );
```
Which will return true if either target_repetitions or repetition_count_callback is set, but not both.

Reply:
TIL; This is pretty cool!